tests/test_integration.py

```diff
@@ -1,13 +1,14 @@
 import dotenv
 from os import getenv
+from pathlib import Path
 import pytest
 import unittest.mock
 
 import todo_app.app
 
 @pytest.fixture
 def client():
-    file_path = dotenv.find_dotenv('.env.test')
+    file_path = Path.cwd().joinpath(".env.test")
```

Comment:
Could get `dotenv.find_dotenv` to work in Docker. It kept failing with "path not found", even though `.env.test` was copied properly. Had to resort to using this semi-hardcoded path.